### PR TITLE
Improve profile functionalities be able to show mulitiple layers by r…

### PR DESCRIPTION
**Pull request by Huyeng**
- The mapper or editor can add the layers from listed layer as creating the map post.
- Can add multiple layers if they have the same source or a table of cartodb. For example: ELCs has a spreadsheet that can produce several map layers such downsize, revoke and so on.
- Fixed the legend issue that always happen caused by cartodb api
- Present the map be consistent a cross the website.
  Note that: all the map requires to has the map_id as before.

Screenshot:
1: input
screenshot_10

screenshot_14

2: results
screenshot_16

screenshot_15

screenshot_19

### CODE REVIEW

**Comment by Huyeng** [comment]
There are some updated code in the theme, they are related, so both the updated code on profile plugin and theme have to update together.

File: post-types/profile-pages.php [hunk 1]
```diff
@@ -123,8 +123,6 @@ public function resource_settings_box($post = false)
       {
           $full_width_middle_content = get_post_meta($post->ID, '_full_width_middle_content', true);
           $full_width_middle_content_localization = get_post_meta($post->ID, '_full_width_middle_content_localization', true);
-          $map_visualization_url = get_post_meta($post->ID, '_map_visualization_url', true);
```
Comment (acorbi) [inline post-types/profile-pages.php]: @Huyeng I have removed the references to the cartodb js api URL

File: post-types/profile-pages.php [hunk 2]
```diff
@@ -154,16 +152,6 @@ public function resource_settings_box($post = false)
               ?></p>
               </td>
               </tr>
-  						<tr>
-   					  <th><label for="_map_visualization_url"><?php _e('CartoDB JSON URL (English)', 'odm');
```
Comment (acorbi) [inline post-types/profile-pages.php]: @Huyeng I have removed the references to the cartodb js api URL

File: post-types/profile-pages.php [hunk 3]
```diff
@@ -195,24 +183,14 @@ public function resource_settings_box($post = false)
   			 	<table class="form-table form-table-localization resource_settings_box">
   		 			<tbody>
               <tr>
-              <th><label for="_full_width_middle_content_localization"><?php _e('Full width content (('.odm_language_manager()->get_the_language_by_site().')', 'odm');
```
Comment (acorbi) [inline post-types/profile-pages.php]: @Huyeng removed duplicated parenthesis

File: post-types/profile-pages.php [hunk 4]
```diff
@@ -195,24 +183,14 @@ public function resource_settings_box($post = false)
...
+              <th><label for="_full_width_middle_content_localization"><?php _e('Full width content ('.odm_language_manager()->get_the_language_by_site().')', 'odm');
               ?></label></th>
               <td>
               <textarea name="_full_width_middle_content_localization" style="width:100%;height: 50px;" placeholder=""><?php echo $full_width_middle_content_localization; ?></textarea>
               <p class="description"><?php _e('Any content can add to under the Editor content and sidebar and  full width of website even the iframe.', 'odm');
               ?></p>
               </td>
-              </tr>
-  						<tr>
-  						 <th><label for="_map_visualization_url_localization"><?php _e('CartoDB JSON URL ('.odm_language_manager()->get_the_language_by_site().')', 'odm');
```
Comment (acorbi) [inline post-types/profile-pages.php]: @Huyeng I have removed the references to the cartodb js api URL

File: post-types/profile-pages.php [hunk 5]
```diff
@@ -459,14 +437,6 @@ public function save_post_data($post_id)
                     update_post_meta($post_id, '_full_width_middle_content_localization', $_POST['_full_width_middle_content_localization']);
                 }
 
-                if (isset($_POST['_map_visualization_url'])) {
```
Comment (acorbi) [inline post-types/profile-pages.php]: @Huyeng I have removed the references to the cartodb js api URL

**Pull request merged**